site/src/api/typesGenerated.ts

```diff
@@ -1202,7 +1202,7 @@ export interface TemplateExample {
 
 // From codersdk/organizations.go
 export interface TemplateFilter {
-  readonly OrganizationID: string;
+  readonly q?: string;
```

Comment:
Is this type missing the organization ID prop? I see it still there on https://github.com/coder/coder/pull/13784/files#diff-dbdae237ab2279354898d4a64bb02289c3b960ba644849f6fc96b15228b603d0R367.

Reply:
In organizations.go, I believe typescript:"-" excludes OrganizationID from the generated types